### PR TITLE
add audio device selection to meetings

### PR DESCRIPTION
**Pull request by EzraEllette**
## description
Allows users to select the audio devices for a meeting.
Does not affect summary or person identification.
related issue: #553

## type of change
- [ ] bug fix
- [x] new feature
- [ ] breaking change
- [ ] documentation update

## how to test
1. check / uncheck devices
2. verify transcript change
3. 

## checklist
- [x] i have read the [CONTRIBUTING.md](https://github.com/mediar-ai/screenpipe/blob/main/CONTRIBUTING.md) file 
- [x] i have added the custom cursor AI prompt to my settings as mentioned in CONTRIBUTING.md and used to write this PR
- [x] my code follows the project's style guidelines
- [x] i have performed a self-review of my code
- [x] i have updated the documentation if necessary
- [x] my changes generate no new warnings
- [ ] i have added tests that prove my fix is effective or that my feature works
- [x] all tests pass locally with my changes
/claim 553

### CODE REVIEW

**Comment by louis030195** [comment]
Screenshot 2024-10-22 at 08 17 37

not sure what happened i only have 1 and 18 (even on main) -

File: screenpipe-app-tauri/components/meeting-history.tsx [hunk 1]
```diff
@@ -267,7 +279,9 @@ export default function MeetingHistory() {
         },
         {
           role: "user" as const,
-          content: `${enhancedPrompt}:\n\n${meeting.fullTranscription}`,
+          content: `${enhancedPrompt}:\n\n${meeting.segments
```
Comment (louis030195) [inline screenpipe-app-tauri/components/meeting-history.tsx:282]: i think it's important to keep the timestamp (probably in a more human readable format) both for human and machine summary

Comment (EzraEllette) [reply screenpipe-app-tauri/components/meeting-history.tsx:282]: Okay I'll update that

Comment (louis030195) [reply screenpipe-app-tauri/components/meeting-history.tsx:282]: image
eg this but more human readable date and probably should tell the LLM current date and stuff about local / UTC or timezone if useful

Comment (EzraEllette) [reply screenpipe-app-tauri/components/meeting-history.tsx:282]: I'll make sure it is the same format

Comment (EzraEllette) [reply screenpipe-app-tauri/components/meeting-history.tsx:282]: image

**Pull request merged**